package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-a11y",
-  "version": "0.1.12",
+  "version": "0.2.0",
```

Comment:
I rely on `ember-release` for this, we can do it separately. Also, if it doesn't change compatibility, I'm going to aim for `0.1.13` so that hat versions work.